handlers/graph.go

```diff
@@ -217,13 +238,13 @@ func buildNamespaceTrafficMap(namespace string, o options.Options, client *prome
 
 		// fetch the internally originating request traffic time-series
 		intIstioVector := promQuery(query, time.Unix(o.QueryTime, 0), client.API())
-		populateTrafficMap(trafficMap, &intIstioVector, o)
+		populateTrafficMap(trafficMap, &intIstioVector, o, "")
 	}
 
 	return trafficMap
 }
 
-func populateTrafficMap(trafficMap graph.TrafficMap, vector *model.Vector, o options.Options) {
+func populateTrafficMap(trafficMap graph.TrafficMap, vector *model.Vector, o options.Options, tcpMetric string) {
```

Comment:
I think this would be nicer as two functions without the conditional logic:
```
populateTrafficMapHttp
populateTrafficMapTcp
```

Reply:
Done!